flutter_hacker_news/lib/router/navigation_constants.dart

```diff
@@ -0,0 +1,10 @@
+///Root router
+const String ROOT_TO_SPLASH = "/";
+const String ROOT_TO_CONTENT = "/content";
+
+///HN NEWS
+const String TAB_HN_NEWS_TO_ROOT = "/";
+const String TAB_HN_NEWS_TO_DETAIL = "/tabHNNewsToDetail";
```

Comment:
the detail of jobs item doesn't exist, there is opening to a custom-tab directly instead native window.